src/lib/Scenes/Favorites/Components/Artists/Relay/ArtistsRenderer.tsx

```diff
@@ -8,14 +8,14 @@ export default ({ render }) => {
   return (
     <QueryRenderer
       environment={environment}
-      query={graphql`
+      query={graphql.experimental`
```

Comment:
@orta - can you explain why this `.experimental` is used in this context?

Reply:
I might not of actually needed it - whenever use you an `@attribute` they need to be using the relay experimental tag instead of the normal graphql tag - an earlier version may have had this, then removed it

Reply:
In Relay v1.3 the directives to define arguments for a fragment we’re still considered experimental, however in current versions of Relay that’s no longer the case. They jut wanted to have an API that would allow them to try things but still being clear the API/feature wasn’t decided upon yet.

Reply:
Oh wait, I now get the question about “in this context”, I don’t think it’s necessary here